dotCMS/src/main/java/com/dotcms/rest/api/v1/system/ConfigurationHelper.java

```diff
@@ -82,8 +82,12 @@ private ConfigurationHelper() {
 						message("notifications_dismiss", locale), // Dismiss
 						message("notifications_dismissall", locale), // Dismiss all
 						this.getRelativeTimeEntry(locale)
-				)
-				);
+				),
+				DOTCMS_PAGINATOR_ROWS,
```

Comment:
In these properties, I think we can replace the word "PAGINATOR" with "PAGINATION".